doc-content/enterprise-only/openshift/registry-auth-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='registry-auth-proc']
+= Configuring OpenShift registry authentication
+
+You must configure OpenShift registry authentication to access Red Hat product images. 
```

Comment:
Maybe add a little more here, like After you do BLAH, you must cofigure..

And I would put the next line in a note.

Any prereqs? Like You must have BLAH installed.. Whatever...

Reply:
It's a prereq itself. There is no "after you do BLAH", the only prereq for this is OpenShift itself, which is a prereq for the entire doc to start with.